lib/modules/user-form-widgets/views/includes/modal-submission-overview.html

```diff
@@ -13,51 +13,12 @@ <h3 class="content-modal-title">
         <thead>
           <tr>
             <th>Inzending ID</th>
-            <th>User ID</th>
-            <th>Mening</th>
-            <th class="sorttable_alpha">IP</th>
+            <th>Ingestuurde gegevens</th>
             <th>Datum</th>
-            <th>OK</th>
           </tr>
         </thead>
         <tbody>
-          <!--{% for vote in data.ideaVotes %}
-            <tr class="{{'rejected' if vote.checked === false}}">
-              <td>{{vote.id}}</td>
-              <td>{{vote.userId}}</td>
-              <td>{{vote.opinion}}</td>
-              {% if showConfirmed %}
-                <td>{{vote.confirmed}}</td>
-              {% endif%}
-              {% if showPostcode %}
-              <td>{{vote.user.zipCode}}</td>
-              {% endif%}
-              {% if showEmail %}
-                <td>{{vote.user.email}}</td>
-              {% endif%}
-              <td>
-                {# was an issue that voting got ip of CMS#}
-                {% if vote.ip === '185.110.174.172' %}
-                  Onbekend
-                {% else %}
-                  {{ vote.ip }}
-                {% endif %}
-              </td>
-              <td>{{vote.createdAt | date('YYYY-MM-DD HH:mm:ss')}}</td>
-              <td class="checked" data-vote="{{vote.id}}" sorttable_customkey="{{vote.checked}}">
-                <form method="get" class="vote-toggle-form" action="/api/site/{{data.global.siteId}}/vote/{{vote.id}}/toggle">
-                  {#
-                  <b {% if vote.checked === false %} style="display: none" {% endif%} class="vote-approved-text"> yes </b>
-                  <b {% if vote.checked !== false %} style="display: none" {% endif%} class="vote-unapproved-text"> no </b>
-                  #}
-                  <button type="submit" class="link" style="background: none; border: 0; padding: 0">
-                    <small {% if vote.checked === false %} style="display: none" {% endif%} class="vote-approved-text">✗</small>
-                    <small {% if vote.checked !== false %} style="display: none" {% endif%} class="vote-unapproved-text">✓</small>
-                  </button>
-                </form>
-              </td>
-            </tr>
-          {% endfor %}-->
+
```

Comment:
Don't we have to show something here?

Reply:
This data is loaded in with AJAX. See user-form-widgets/public/js/main.js line 53:

```
$.get('/modules/user-form-widgets/submissions?form=' + formId)
                    .done(function (data) {
                        var rows = [];
                        $(data).each(function () {
                            var submittedData = '';
                            
                            $.each(this.submittedData, function (k, v) {
                                if (v.indexOf('http') !== -1) {
                                    v = '<a href="' + v + '" target="_blank">' + v + '</a>';
                                }
                                submittedData += '<strong>' + k + '</strong>:' + v + '<br />';
                            });
                            
                            rows.push([this.id, submittedData, this.createdAt]);
                        });
                        
                        dataTable.rows.add(rows).draw();
                    })
                    .fail(function (err) {
                        console.error('ajax', err);
                    });
```